changelog.d/3578.bugfix

```diff
@@ -0,0 +1 @@
+Added "media-src: 'self'" to CSP for resources downloading
```

Comment:
sorry, that doesn't mean a lot to me. How about: "Fix problem when playing media from Chrome" (if that is an accurate description)?

Reply:
Yes it’s much better.